test/e2e/istio_test.go

```diff
@@ -0,0 +1,212 @@
+// +build istio
+
+package e2e
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"k8s.io/apimachinery/pkg/labels"
+
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	log "github.com/sirupsen/logrus"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type IstioTestSuite struct {
+	suite.Suite
+}
+
+// LIFECYCLE - Suite
+func (suite *IstioTestSuite) SetupSuite() {
+	t = suite.T()
+	var err error
+	ctx, err = prepare(t)
+	if err != nil {
+		if ctx != nil {
+			ctx.Cleanup()
+		}
+		require.FailNow(t, "Failed in prepare")
+	}
+	fw = framework.Global
+	namespace = ctx.GetID()
+	require.NotNil(t, namespace, "GetID failed")
+
+	// label namespace
+	ns, err := framework.Global.KubeClient.CoreV1().Namespaces().Get(context.Background(), namespace, metav1.GetOptions{})
+	if err != nil {
```

Comment:
In general in tests I'd prefer to see require.NoError() (or another appropriate require function called.) unless you're going to actually try to recover from the error.  If not, require.NoError is more concise and gives a nice short stack trace.

Reply:
Fixed